### ISSUE TITLE
dataset

### ISSUE THREAD

**Issue by YangYangTaoTao** [markendo/GaitForeMer#1]
What a great pity that the dataset isn't open source.

**Comment by asy51** [comment]
indeed. Does anyone know of any alternative sources for this data and/or related datasets that are publicly available?
